File: apenas java/idadeEmDias.py
dia = int(input())
diafinal = dia % 365
ano = int((dia - diafinal)/365)
diafinal = diafinal % 30
mes = int((dia - ano * 365 - diafinal)/ 30)
print('{} ano(s)\n{} mes(es)\n{} dia(s)'.format(ano, mes, diafinal))

File: apenas java/idadeEmDias.cpp
#include <iostream>
using namespace std;
int main() {
    int dia;
    cin >> dia;
    int ano = 0;
    while (dia - 365 >= 0) {
        ano++;
        dia -= 365;
    }
    int mes = 0;
    while (dia - 30 >= 0) {
        mes++;
        dia -= 30;
    }

    cout << ano << " ano(s)" << endl;
    cout << mes << " mes(es)" << endl;
    cout << dia << " dia(s)" << endl;

    return 0;
}